tests/functional/event_handler/_pydantic/test_openapi_params.py

```diff
@@ -105,7 +105,7 @@ def test_openapi_with_custom_params():
     def handler(
         count: Annotated[
             int,
-            Query(gt=0, lt=100, examples=[Example(summary="Example 1", value=10)]),
+            Query(gt=0, lt=100, examples=[10]),
```

Comment:
I would prefer not to change this test, but rather add a new one with the new accepted types like int, boolean, float and others.